### ISSUE TITLE
chart.update line markers not showing when accessibility module included

### ISSUE THREAD

**Issue by kai2468** [highcharts/highcharts#12718]
#### Expected behaviour
New line series should have their markers shown on updating chart

#### Actual behaviour
It doesnt.

#### Live demo with steps to reproduce
https://jsfiddle.net/2745rxdt/1/

Push the "Click Me" button, new series markers are not visible. If you remove the accessibility module though, it seems to work fine.

#### Product version
Latest

@oysteinmoseng

**Comment by oysteinmoseng** [comment]
Reproducible without data module: https://jsfiddle.net/s52vf8zd/

Probably related to forced marker logic, I will take a look.

**Comment by oysteinmoseng** [comment]
Actually seems to happen because of dummy elements for null points. The dummy elements are hidden, and the hidden styling is not overwritten when the point receives an actual element instead.
